controllers/managedocs_controller_test.go

```diff
@@ -283,6 +284,94 @@ var _ = Describe("ManagedOCS controller", func() {
 			})
 		})
 
+		When("the sds count is increased in secret", func() {
+			It("should increase storage cluster size", func() {
+				ctx := context.Background()
+
+				managedOCS := &v1.ManagedOCS{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      ManagedOCSName,
+						Namespace: TestNamespace,
+					},
+				}
+				managedOCSKey := utils.GetResourceKey(managedOCS)
+				Expect(k8sClient.Get(ctx, managedOCSKey, managedOCS)).Should(Succeed())
+
+				sc := &ocsv1.StorageCluster{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      storageClusterName,
+						Namespace: TestNamespace,
+					},
+				}
+				scKey := utils.GetResourceKey(sc)
+				Expect(k8sClient.Get(ctx, scKey, sc)).Should(Succeed())
+
+				secret := &corev1.Secret{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      TestAddOnParamsSecretName,
+						Namespace: TestNamespace,
+					},
+					Data: map[string][]byte{
+						"size": []byte("2"),
```

Comment:
We do not support a value of 2, only 1 or 4. I suggest using 1 and upscaling to 4